Add Selected wave csv export support

dlsloan opened the pull request.

dlsloan committed:
csv: selected waves csv export

Export selected waves to csv file with or without timestamps

dlsloan committed:
Increment eid-version and update change log

jeskritt reviewed (changes requested):
It's borked. I tried to time series CSV export istrm6 bus of attached waveforms

[gtkwave_csv_break.zip](https://github.com/user-attachments/files/15930501/gtkwave_csv_break.zip)

dlsloan committed:
csvexport: Skip null traces

Groups in waveform windows can be selected and contain NULL
bitvector/nodeptr data. Skip these when running csv export logic to
prevent segv's.

dlsloan commented:
Completes #2
